snmp/tests/conftest.py

```diff
@@ -15,6 +15,8 @@
 FILES = [
     "https://ddintegrations.blob.core.windows.net/snmp/f5.snmprec",
     "https://ddintegrations.blob.core.windows.net/snmp/3850.snmprec",
+    # Todo: move meraki snmprec to common storage bucket
+    "https://gist.githubusercontent.com/ls339/e6a0d28a0595783819d094b9d1dccd84/raw/2978e3ab48aae46aa4fe64a305caeb71d02934eb/meraki-cloud-controller.snmprec",  # noqa: E501
```

Comment:
@therve What is the process of getting the snmprec file on `/ddintegrations.blob.core.windows.net` ?

Reply:
If that's the same file as your gist, you can directly put in the `data` directory as it's small enough.

Reply:
@ls339 https://github.com/DataDog/integrations-core/tree/master/snmp/datadog_checks/snmp/data/profiles